lib/vdsm/api/vdsm-api.yml

```diff
@@ -8451,6 +8451,21 @@ types:
             name: path
             type: string
 
+        -   description: The UUID of the Image
+            name: imageID
+            type: *UUID
+            added: '4.5'
+
+        -   description: The UUID of the Storage Domain containing the Volume
+            name: domainID
+            type: *UUID
+            added: '4.5'
+
+        -   description: The UUID of the Volume
+            name: volumeID
+            type: *UUID
+            added: '4.5'
```

Comment:
Please check the order of the properties in BackupDisk:

        -   description: The UUID of the storage domain
            name: sd_id
            type: *UUID
        -   description: The UUID of the image
            name: img_id
            type: *UUID
        -   description: The UUID of the volume
            name: vol_id
            type: *UUID

This order is not accidental - we have a name for these ids - `DIV` - domain, image, volume. We want to keep this order everywhere.

Please update the *ScatchDisk types to have the standard order.

Additionally, we are missing a fix for BackupDisk keys - the actual API (what engine
is sending) is domainID, imageID, volumeID, so we should replace sd_id, img_id, vol_id 
with the actual names.

This can be done in another PR, and opening new issue for this will be good so we
don't forget to do this.